nanofi/tests/CSite2SiteTests.cpp

```diff
@@ -17,6 +17,7 @@
  */
 
 #include <stdlib.h>
+#include <signal.h>
```

Comment:
This should be wrapped in `#ifndef WIN32` as well

Reply:
fixed